resources/assets/components/Tags/index.js

```diff
@@ -60,7 +65,7 @@ class Tags extends React.Component {
         <ul className="aligned-actions">
           {map(tags, (label, key) => (
             <li key={key}>
-              <Tag isActive={this.props.tagged.includes(key)} isClicked={this.props.onTag} label={label} post={this.props.id} />
+              <Tag isActive={this.props.tagged.includes(key)} isClicked={this.props.onTag} label={label} post={this.props.id} disable={this.props.disableTags} />
```

Comment:
Could we renamed this prop to `disabled` so it's clearer it's a boolean, and not an event handler?